src/Interfaces.ts

```diff
@@ -1,24 +1,40 @@
 export interface Claim<T extends ClaimAttributes = ClaimAttributes> {
+  readonly '@context'?: any
   readonly id?: string
-
   readonly publicKey?: string
   readonly signature?: string
-  readonly dateCreated?: Date
-
+  readonly created?: string
   readonly type: ClaimType
   readonly attributes: T
 }
 
+interface ClaimContext {
+  readonly '@context': {
+    readonly [key: string]: string
+  }
+}
+
 export function isClaim(object: any): object is Claim {
   // TODO: use joi or protobuf
-  return (
-    object.id &&
-    object.publicKey &&
-    object.signature &&
-    object.type &&
-    object.attributes &&
-    object.dateCreated instanceof Date
-  )
+  return !!(object.id && object.publicKey && object.signature && object.type && object.attributes && object.created)
+}
+
+// WARNING: This MUST account for ALL of the attributes in a Claim, except for id, @context, and signature.
+// Otherwise those attributes without context will be left out of the canonized/signed claim.
+// Refer to https://www.w3.org/2018/jsonld-cg-reports/json-ld/#the-context
+export const ClaimContext: ClaimContext = {
+  '@context': {
+    attributes: 'http://schema.org/CreativeWork',
+    author: 'http://schema.org/author',
+    text: 'http://schema.org/text',
+    created: 'http://purl.org/dcterms/created',
```

Comment:
This is giving a 503 error, it may be correct though, just pointing it out.

Reply:
ahhh you are correct, it should be http://purl.org/dc/terms/created
 per http://dublincore.org/documents/dcmi-terms/#terms-created